### PR TITLE
fix(ports): adding info tooltip and reset the value

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-587)

Reset the value when toggle public and adding a tooltip on both places we have a port creation.

image

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z4abd77e0-z9c6f63b1-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z4abd77e0-z1bdae0e3-gtw.za8ad0659.bool.sh)

File: libs/shared/ui/src/lib/components/inputs/input-text/input-text.tsx [hunk 1]
```diff
@@ -35,8 +35,9 @@ export function InputText(props: InputTextProps) {
   const [currentType, setCurrentType] = useState(type)
 
   useEffect(() => {
+    console.log(value)
```
Comment (RemiBonnet) [inline libs/shared/ui/src/lib/components/inputs/input-text/input-text.tsx:38]: You can remove it !

**Review by RemiBonnet — COMMENTED** [review]
Well done @bdebon !! 🎉 , small comments: 

- Can you rename the PRs with fix or feat prefix ? Because it's not an chore improvement 
- I see several possible improvements for this screen 
Capture d’écran 2022-11-24 à 10 22 46

   - The toggle should be "small"  
   - I think you can remove the `index` for the row you can have only "Application port" and "External port" with lowercase port 
   - The tooltip could be on the input "External port" (like modal)

File: libs/pages/services/src/lib/ui/page-application-create/page-application-create-port/port-row/port-row.tsx [hunk 2]
```diff
@@ -45,14 +46,19 @@ export function PortRow(props: PortRowProps) {
             className="flex-grow"
             name={field.name}
             onChange={field.onChange}
-            value={field.value}
+            value={externalPortWatch} // passing a watch here because setValue with undefined does not work: https://github.com/react-hook-form/react-hook-form/issues/8133
```
Comment (RemiBonnet) [inline libs/pages/services/src/lib/ui/page-application-create/page-application-create-port/port-row/port-row.tsx:49]: :'(

**Pull request merged**